README.md

```diff
@@ -11,6 +11,8 @@ First time to Cromwell? Get started with [Tutorials](http://cromwell.readthedocs
 
 Thinking about contributing to Cromwell? Get started by reading our [Contributor Guide](CONTRIBUTING.md).
 
+Cromwell has a growing ecosystem of community-backed projects to make your experience even better! Check out our [Ecosystem](https://cromwell.readthedocs.io/en/develop/Ecosystem/) page to learn more.
```

Comment:
If you can, I would make this a relative rather than absolute link (right now it hard-codes the `develop` version of the docs

Reply:
Great! 

To address your comment, I tried to follow the convention of the other two links in the README which point to `develop`. I'd recommend we update all of them to the same thing, but I'm not sure what you had in mind about making the links relative (the link to `CONTRIBUTING.md` is relative, but that's because the intent is to land them on the actual markdown file).

Reply:
Oh, sorry, my bad, I thought this was already in a readthedocs page.

I'd probably point at the `stable` rather than `develop` docs in that case (but have to accept that it won't actually be a live link until the next Cromwell version is published).

Reply:
Okay, would you like me to update all the links to `stable` so they are in sync? Should be easy enough.

Reply:
Sure, and thank you!